doc/source/ray-air/examples/pytorch_tabular_starter.py

```diff
@@ -33,27 +33,16 @@
 import numpy as np
 import pandas as pd
 
-from ray.data.preprocessors import BatchMapper, Chain
+from ray.data.preprocessors import BatchMapper, Chain, Tensorizer
 
 # Get the training data schema
 schema_order = [k for k in train_dataset.schema().names if k != "target"]
```

Comment:
Remove this from example.

Reply:
It will break though? See below comment.